amico/core.py

```diff
@@ -261,8 +272,11 @@ def load_kernels( self ) :
         # auxiliary data structures
         idx_OUT, Ylm_OUT = amico.lut.aux_structures_resample( self.scheme, self.get_config('lmax') )
 
+        # hash table
+        self.htable = amico.lut.load_precomputed_hash_table( self.get_config('ndirs') )
+
         # Dispatch to the right handler for each model
-        self.KERNELS = self.model.resample( self.get_config('ATOMS_path'), idx_OUT, Ylm_OUT, self.get_config('doMergeB0') )
+        self.KERNELS = self.model.resample( self.get_config('ATOMS_path'), idx_OUT, Ylm_OUT, self.get_config('doMergeB0'), self.get_config('ndirs') )
```

Comment:
Function non compatible with files created with previous versions. 
When it is run with files created with previous versions (1.0.*) produces the error: 
`ValueError: shapes (360,364) and (181,364) not aligned: 364 (dim 1) != 181 (dim 0)`
on amico/lut.py:293 in function resample_kernel. 
The kernels need to be regenerated. The error can be cached and if it is related to the previous number of directions 181  show a message like 

> [ LUT computed with a previous version of AMICO. Call "generate_kernels( regenerate=True )" to update the LUT. ]